node/src/database/config_dumper.rs

```diff
@@ -195,6 +195,7 @@ mod tests {
             other => panic!("Was expecting Value::Object, got {:?} instead", other),
         };
         let expected_value = json!({
+           "chainName": "ropsten".to_string(),
```

Comment:
this line could be:
` "chainName": TEST_DEFAULT_CHAIN_NAME.to_string(),`
as used above on line 178.